frontend/packages/console-app/locales/en/console-app.json

```
@@ -421,12 +421,17 @@
   "Container runtime": "Container runtime",
   "Kubelet version": "Kubelet version",
   "Kube-Proxy version": "Kube-Proxy version",
+  "CPU Architecture": "CPU Architecture",
```

```suggestion
  "CPU architecture": "CPU architecture",
```

Per PF guidelines, this should be sentence case

---

Thanks for the comments , modified as suggested above .